Sources/NIOHTTP1/HTTPDecoder.swift

```diff
@@ -113,6 +113,9 @@ private class BetterHTTPParser {
         }
         self.settings.pointee.on_message_complete = { opaque in
             BetterHTTPParser.fromOpaque(opaque).didReceiveMessageCompleteNotification()
+            // Temporary workaround for https://github.com/nodejs/llhttp/issues/202, should be removed
+            // when that issue is fixed.
+            opaque?.pointee.content_length = 0
```

Comment:
Can we track this in one of our GH issues as well?

Reply:
Can we repurpose #2274 to track this?

Reply:
We sure can!